core/runtime/impl/wasm_memory_impl.cpp

```diff
@@ -148,5 +155,11 @@ namespace kagome::runtime {
                                 const std::array<uint8_t, 16> &value) {
     set<std::array<uint8_t, 16>>(addr, value);
   }
+  void WasmMemoryImpl::storeBuffer(kagome::runtime::WasmPointer addr,
+                                   const kagome::common::Buffer &value) {
+    const auto &value_vector = value.toVector();
+    memory_.insert(memory_.begin() + addr, value_vector.begin(),
```

Comment:
Please, check that `memory_.begin() + addr` is not out of bounds

Reply:
left todo for now